daemon/containerd/image.go

```diff
@@ -68,14 +68,17 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options ima
 		exposedPorts[nat.Port(k)] = v
 	}
 
+	derefTimeSafely := func(t *time.Time) time.Time {
+		if t != nil {
+			return *t
+		}
+		return time.Time{}
```

Comment:
Should we fixate `time.Time()` either at the start of the loop, or before looping? (it looks like currently the history item and config both call this function, so would end up with different timestamps?

Reply:
🤦 for some reason I read `time.Time{}` as `time.Now()`, and thought we would be generating two timestamps